recipe/meta.yaml

```diff
@@ -3,20 +3,19 @@ package:
   name: salib
   version: {{ version }}
 source:
-  fn: salib-{{ version }}.tar.gz
-  url: https://github.com/SALib/SALib/archive/v{{ version }}.tar.gz
-  sha256: d754a6199e116014f268a2e5844b28f6ddbf8ceb69156a9974c5833ba1d37f17
+  fn: Salib-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/SALib/SALib-{{ version }}.tar.gz
```

Comment:
The url becomes valid when using an S instead of a p between source and SALib:

https://pypi.io/packages/source/S/SALib/SALib-{{ version }}.tar.gz

Reply:
Thanks @dgdekoning I'll try that now